core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/GroupDataBinderImpl.java

```diff
@@ -207,16 +207,43 @@ public void create(final Group group, final GroupTO groupTO) {
             group.setName(groupPatch.getName().getValue());
         }
 
+        PropagationByResource<String> propByRes = new PropagationByResource<>();
         // owner
         if (groupPatch.getUserOwner() != null) {
-            group.setUserOwner(groupPatch.getUserOwner().getValue() == null
-                    ? null
-                    : userDAO.find(groupPatch.getUserOwner().getValue()));
+            if (StringUtils.isBlank(groupPatch.getUserOwner().getValue())) {
```

Comment:
Checking for `null` equality should be enough here (rather than `StringUtils#isBlank` since the expected value is an user key, e.g. UUID

Reply:
agree, done